tests/test_utils.py

```diff
@@ -162,8 +162,8 @@ def test_power_kramers_kronig():
     assert 1540 == power_law_kramers_kronig(1, 1, 1540, 1, 2.5)
     assert 1540 == power_law_kramers_kronig(1, 1, 1540, 1, 1)
     with pytest.warns(UserWarning):
-        ans = power_law_kramers_kronig(1, 1, 1540, 1, 4)
-        assert ans == 1540
+        c = power_law_kramers_kronig(1, 1, 1540, 1, 4)
```

Comment:
Isn't it a bit odd that it not only fixes typing errors but also renames variables? Not sure if we want this? Does it check if there is already a variable with the same name?

Reply:
I'm more interested about how it came up with variable name `c`? :)

Reply:
it renamed it to and and I changed it to c.